testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -571,5 +571,36 @@ export default function() {
             const value = prepareEmbedValue(this.$element.find(`.${MENTION_CLASS}`).parent().html());
             assert.strictEqual(value, expectedMention);
         });
+
+        test('Mentions template must be applyed after removing second editor(T1110266)', function(assert) {
```

Comment:
I don't understand this name. Why this template **must** be applied after other editor remove?

```suggestion
        test('Mention template is applied correctly after removing of a neighbour editor with the same mention marker(T1110266)', function(assert) {
```